### PR TITLE
Navbar with bootstrap done

### PR DESCRIPTION
**Pull request by JD235**
With complete use of bootstrap Spacing Utilities.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
         <!-- Navbar Starts -->
         <header class="header-menu" id="head_top">
             <div class="main-menu" style="margin-top: 9%;">
-                <nav class="navbar fixed-top navbar-expand-lg navbar-light " style="background-color: white;">
+                <nav class="navbar fixed-top navbar-expand-lg navbar-light pb-xl-0 pb-lg-0 pb-md-0 pb-sm-0 pb-0">
                     <a class="navbar-brand" href="#"><img class="img" src="./assets/mozguj-horizontal-black.png"
                             alt="user" height="100px" width="210px"></a>
```
Comment (arps18) [inline index.html]: Avoid Height width in HTML itself!

**Review by arps18 — COMMENTED** [review]
Do the necessary changes!

File: index.html (hunk 1 above)
Comment (Amorpheuz) [reply index.html]: Along with the core high-res image, I would suggest you resize a copy of it to fit the actual navbar dimensions. No point in sending the full-res image for such a small-sized requirement.

Comment (JD235) [reply index.html]: changes have done please do check it once.

**Review by Amorpheuz — CHANGES_REQUESTED** [review]

**Review by Amorpheuz — APPROVED** [review]

**Review by Amorpheuz — CHANGES_REQUESTED** [review]
I think you have mistakenly added the `.DS_Store` files, do get rid of them before merging. Rest looks good to be merged.

**Pull request merged**